tags/network_architecture.yaml

```diff
@@ -199,3 +199,5 @@ values:
     display_name: word2vec skip-gram
   - id: wrn-106
     display_name: WRN-106
+  - id: wav2vec2-base
```

Comment:
Is there a non-base version? My preference would be to call this `wav2vec2`, and we can distinguish future versions as needed, eg `wav2vec2-v2`, etc.